src/exercises/bare-metal/compass.md

```diff
@@ -0,0 +1,19 @@
+# Compass
+
+We will read the temperature from an I2C compass, and log the readings to a serial port.
+
+Hint: check the documentation for the [`lsm303agr`](https://docs.rs/lsm303agr/latest/lsm303agr/) and
+[`microbit-v2`](https://docs.rs/microbit-v2/latest/microbit/) crates.
+
+If you have time, try displaying it on the LEDs somehow too, or use the buttons somehow.
```

Comment:
```suggestion
If you have time, try making the LED in the direction of North light up.
```

Reply:
I'd like to leave it up to the student to play around and see how they'd like to use the LEDs and buttons here, rather than being too specific, seeing as it's an extension anyway.

Reply:
Why make it up to the students when we can give them a concrete task? In my experience, people are not super good at coming up with these ideas by themselves :smile: People will have just seen the board and will only have a tiny bit of context. For that reason, I try to avoid being vague in the suggestions.

Reply:
Okay, the first part is concrete for sure. But everything we write down becomes instructions for someone — either in your class or on the Internet. So that's why I'm suggesting to avoid vague language where possible.

It's not a big deal, I just thought it was easy to make the extra task concrete as well. My thinking is that the focus of the class is on using the microcontroller — finding things to do with the microcontroller should be our task and we should not leave it to the reader.